src/components/Header/Header.module.scss

```diff
@@ -0,0 +1,107 @@
+.header {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 24px 370px;
+
+    position: relative;
+    width: 1920px;
+    height: 88px;
+    
+    /* Primaty/Grey Light */
+    background: #F6F5F4;
```

Comment:
If this color refers from our design system(theme CSS), what if the theme CSS file changes the color value of grey light?

Reply:
https://github.com/taodemy/taotalk/blob/457af72abe43f54f6b3c079e56f580a15ec67f7f/src/components/Header/Header.module.scss#L1
https://github.com/taodemy/taotalk/blob/457af72abe43f54f6b3c079e56f580a15ec67f7f/src/components/Header/Header.module.scss#L9-L10
https://github.com/taodemy/taotalk/blob/457af72abe43f54f6b3c079e56f580a15ec67f7f/src/components/Header/Header.module.scss#L33-L34

I have imported color settings from the design system, and replace color in scss file from hard coding to imported variables.